taskify.Server/Controllers/Tasks/TaskPostRequestBody.cs

```diff
@@ -1,14 +1,22 @@
-﻿namespace taskify.Server.Controllers.Tasks
+﻿using System.ComponentModel.DataAnnotations;
+using System.Text.Json.Serialization;
+
+namespace taskify.Server.Controllers.Tasks
 {
     public class TaskPostRequestBody
     {
+        [Required]
         public int AuthorId {  get; set; }
+        [Required]
         public string Title { get; set; }
         public string? Description { get; set; }
         public DateTime? DueTo { get; set; }
+        [Required]
         public int Priority {  get; set; }
+        [Required]
         public int Difficulty { get; set; }
         public string TagName { get; set; }
```

Comment:
wydaje mi się, że '**TagName**' również powinien być wymagany - co sądzisz?

Reply:
Myślę, że powinien być generowany samoczynnie. Wtedy zamieniłbym typ ze stringa na inta i by się generowało automatycznie

Reply:
Ale to chyba dam później, bo teraz mamy ważniejsze rzeczy do roboty

Reply:
racja, zamykam.